OrbitCore/Transaction.h

```diff
@@ -0,0 +1,32 @@
+#ifndef ORBIT_CORE_TRANSACTION_H_
+#define ORBIT_CORE_TRANSACTION_H_
+
+#include <atomic>
+#include <functional>
+#include <string>
+
+#include "Message.h"
+
+namespace orbit {
+
+struct TransactionHandler {
+  MessageType GetType() const { return message_type; }
```

Comment:
> Since the symbols are not sent to the instance anymore, hooking into functions does not work anymore when using local symbols. I don't think it is necessary to have the symbols on the instance, but without them apparently something is missing somewhere. Maybe you find time to look into that.
> 
> There is a crash for me when I load symbols of multiple modules at once, when some of the symbols are locally available and others are on the instance (or not available at all):
> • Click process UE4Game.elf (The UE4Game.debug file is local)
> • select modules UE4Game.elf & libggp.so (no symbols locally for libggp.so)
> • right click -> load symbols
> • crash
> I did not look into why this crashes, but it worked before this PR

Hi @antonrohr , thanks for the info!  My test must have been loading symbols from the instance and not locally, my bad.  I'll make sure this works.